### ISSUE TITLE
Change color of the scroll bar when changing the virtual background

### ISSUE THREAD

**Issue by Sophalso** [jitsi/jitsi-meet#15652]
### What problem are you trying to solve?

Hello,
The color of the scroll bar when changing the virtual background, before going to the conference, is dark and cannot be seen well.
Is it possible to change the color of the scroll bar ?

We use Google Chrome v133.0.6943.127 and Jitsi v2.0.10008.

![Image](https://github.com/user-attachments/assets/e39d57c0-9343-41b8-8b15-c2e702e64166)

### What solution would you like to see?

Change color of the scroll bar

**Comment by saghul** [comment]
I believe that is the native browser scrollbar. We don't style them.

**Comment by manzil-infinity180** [comment]
On macOs

https://github.com/user-attachments/assets/eeee09e1-2f1e-4b1b-bce0-216695cdc71d

here is the video for it 

Let me see if I can change it

**Comment by manzil-infinity180** [comment]
@damencho should we need custom scroll bar for it ! Yep the above video which i shared is from native

**Comment by saghul** [comment]
I wouldn't change it.

**Comment by Sophalso** [comment]
OK, thank you for your reply. We understand.

If possible can we have the background with a customizable color?

![Image](https://github.com/user-attachments/assets/1517bd4d-0d69-41dd-ae12-12549eb851ef)

**Comment by saghul** [comment]
That is the background of all dialogs.

@raduanastase8x8 do you have an opinion? I do agree that the toolbar is barely visible...

**Comment by raduanastase8x8** [comment]
> I believe that is the native browser scrollbar. We don't style them.

Apparently we style them 

![Image](https://github.com/user-attachments/assets/c671cad4-8af2-4372-8720-80d812b24622)

I would go for something like this for the scroll bar

![Image](https://github.com/user-attachments/assets/591271e9-2442-4cff-9bfa-71ca9e493ac0)

**Comment by saghul** [comment]
🤦  My bad!

**Comment by manzil-infinity180** [comment]
@saghul should i raise pr for it ?

**Comment by ishsarin** [comment]
Hi @saghul,
I would like to work on this issue.
